nomos-da/network/core/src/protocols/replication/behaviour.rs

```diff
@@ -177,6 +179,12 @@ impl PendingOutbound {
     }
 }
 
+#[derive(Debug)]
+pub struct ReplicationConfig {
+    pub seen_message_cache_size: usize,
+    pub seen_message_ttl: Duration,
```

Comment:
If this is gonna have a minimum duration it would probably be good to use the utils type for this.

Reply:
Also you may need `Deserialize` for this? How is it loaded otherwise?